internal/app/users/service.go

```diff
@@ -95,30 +95,30 @@ func (us *service) GetIntranetUserData(ctx context.Context, req dto.GetIntranetU
 	url := fmt.Sprintf("%s%s%d", config.IntranetBaseUrl(), constants.GetIntranetUserDataPath, req.UserId)
 	intranetReq, err := http.NewRequest(http.MethodGet, url, nil)
 	if err != nil {
-		logger.Errorf("error in creating new get user request. err: %s", err.Error())
+		logger.Errorf(ctx, "error in creating new get user request. err: %s", err.Error())
 		err = apperrors.InternalServerError
 		return
 	}
 
 	intranetReq.Header.Add(constants.AuthorizationHeader, req.Token)
 	resp, err := client.Do(intranetReq)
 	if err != nil {
-		logger.Errorf("error in intranet get user api. status returned: %d, err: %s  ", resp.StatusCode, err.Error())
-		logger.Errorf("error response: %v", resp)
+		logger.Errorf(ctx, "error in intranet get user api. status returned: %d, err: %s  ", resp.StatusCode, err.Error())
+		logger.Errorf(ctx, "error response: %v", resp)
```

Comment:
can't this be logged with above log statement?

Reply:
updated